src/find/main.rs

```diff
@@ -0,0 +1,9 @@
+extern crate glob;
```

Comment:
Also, the copyright in LICENSE should be to either "Google Inc." or to "the findutils project maintainers".